File: Assignment8/Assignment8_2.c
//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
// file name   : Assignment8_2.c 
// Description : Write a program which accept accept Width and height of rectangle from user and calculate its area
// Author      : Aniket Devaba Markad 
// Date        : 16/05/2025
// 
////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////// 

#include<stdio.h>

////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////// 
// 
// Function Name : RectArea
// Description   : This function is used to calculate of area rectangle.
// Input         : float
// Output        : double
// Author        : Aniket Devaba Markad 
// Date          : 16/05/2025 
// 
///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////// 

double RectArea(float fWidth,float fHeight)
{   
     double Area = 0.0;
     Area = fWidth*fHeight;
     return Area;
}
int main()
{
    float fValue1 = 0.0,fValue2 = 0.0;
    double dRet = 0.0;

    printf("Enter Width");
    scanf("%f",&fValue1);

    printf("Enter height");
    scanf("%f",&fValue2);

    dRet=RectArea(fValue1,fValue2);
    printf("Area of rectangle : %lf",dRet);

    return  0;

}

File: Java/Assignment8/Assignment8_2.java
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
// file name   : Assignment8_2.java
// Description : write a program which Accept width & height of the rectangle from user and calculate its area
// input.      : 5.3  9.78
// output.     : 51.834
// Author      : Aniket Devaba Markad                        
// Date        : 1/1/2026
//
////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

import java.util.Scanner;

class  Number
{

/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
// Function Name : RectangleArea
// Description   : This function is used to calclate the area of rectangle
// Author        : Aniket Devaba Markad 
// Date          : 1/1/2026 
// 
///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

   double RectangleArea(float fwidth ,float fheight)
   {
        double Area = 0.0;
        Area = fwidth * fheight; 
        return Area;
   }
}
public class Assignment8_2
{
    public static void main(String a[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("enter the width ");
        float iValue1 = sobj.nextFloat();

        System.out.println("enter the height ");
        float iValue2 = sobj.nextFloat();

        Number nobj = new Number();
        double iRet = nobj.RectangleArea(iValue1,iValue2);

        System.out.println("area of rectangle is : " +iRet);
    }
}




